arch/arm/boot/dts/overlays/Makefile

```diff
@@ -46,6 +46,12 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	crystalfontz-cfa050_pi_m.dtbo \
 	cutiepi-panel.dtbo \
 	dacberry400.dtbo \
+	devterm-bt.dtbo \
```

Comment:
There needs to be a description of these overlays and any overrides they support in the README file.

Are they applicable to particular hardware? 2711/CM4 only seeing as they use spi4, but it also remaps GPIOs 44&45? Is there a real use case for the individual elements, or is it more sense to have one complete product overlay?